.circleci/config.yml

```diff
@@ -0,0 +1,12 @@
+version: 2
+jobs:
+  build:
+    working_directory: /go/src/github.com/envoyproxy/go-control-plane
+    docker:
+      - image: rshriram/envoy-go-build:956131ed
```

Comment:
can we pin to a generic image?

Reply:
So, the standard go image from circle CI does not have protoc and glide installed. So, i had to create this. I thought that once we land this, we can (a) try to reuse lyft/envoy-build (with some improvements), or push this as a standard build image to dockerhub.